ext/liquid_c/vm_assembler.h

```diff
@@ -158,20 +164,20 @@ static inline void vm_assembler_add_find_variable(vm_assembler_t *code)
 
 static inline void vm_assembler_add_lookup_const_key(vm_assembler_t *code, VALUE key)
 {
-    vm_assembler_increment_stack_size(code, 1);
+    vm_assembler_reserve_stack_size(code, 1); // push 1, pop 2, push 1
```

Comment:
This looks a bit weird here and in `vm_assembler_add_lookup_command`, because we push the value on the stack to fall-through to the case for `OP_LOOKUP_KEY` in the VM switch statement.